osmnx/settings.py

```diff
@@ -32,6 +32,14 @@
                     'highway', 'maxspeed', 'service', 'access', 'area',
                     'landuse', 'width', 'est_width', 'junction']
 
+# tags and attributes for generating an OSM XML file
+osm_node_attrs = [
```

Comment:
maybe name these like `osm_xml_node_attrs` etc to make it a little more explicit for re-use?

Reply:
Done.